packages/ra-ui-materialui/src/layout/Error.tsx

```diff
@@ -53,7 +53,7 @@ function goBack() {
     window.history.go(-1);
 }
 
-const Error = props => {
+const ErrorComponent: FC<ErrorProps> = props => {
```

Comment:
no, we don't name all our components XXXComponent

Reply:
This is done due to the fact that the JS `Error` class is overridden in the context, but is needed as a type for the props.